lib/krumo/class.krumo.php

```diff
@@ -1047,7 +1047,7 @@
 ?>
 <li class="krumo-child">
 
-  <div class="krumo-element<?php echo count($data) > 0 ? ' krumo-expand' : '';?>"
+  <div class="krumo-element<?php echo !empty($data) ? ' krumo-expand' : '';?>">
     <?php if (count($data) > 0) {?> onClick="krumo.toggle(this);"<?php } ?>
     onMouseOver="krumo.over(this);"
```

Comment:
IMO, the whole block should look like:

```PHP
<div class="krumo-element<?php echo !empty($data) ? ' krumo-expand' : '';?>"
    <?php if (!empty($data)) {?> onClick="krumo.toggle(this);"<?php } ?>
```

Reasons:

1. The closing tag `>` on line 1050 you added will break the whole thing, because it duplicating the original closing in the very end of line 1053:

```PHP
onMouseOut="krumo.out(this);">
```

2. Since the lines 50-53 are to output a single HTML code, the same offending variable comparison `count($data) > 0` in line 1051 should be replaced with just `!empty($data)`.

Reply:
Sorry for the late responce @alanmels 

You are correct. I have fixed this.